platform/broadcom/sonic-platform-modules-dell/s5232f/pddf/sonic_platform/chassis.py

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env python
```

Comment:
should it be python or python3 ?

Reply:
Should be Python, ensures that the script is executed with the Python interpreter found in the user’s environment, regardless of where Python is installed on the system.